_includes/scripts.html

```diff
@@ -0,0 +1,4 @@
+<script src="//unpkg.com/hierplane@0.0.5/dist/static/hierplane.bundle.js"></script>
```

Comment:
Let's omit the version (`@0.0.5`) so it always uses the latest.  Otherwise it'll be a PITA to update this file with every version tweak.

It does mean the demos on the site could break.  We'll want to test that as we rev on the library.

Reply:
I'll use the reference called out in the README verbatim once the minified .js is up.

Reply:
Done in latest commit.